recipes-extended/xen/files/tboot-measure-and-launch-from-xen-efi.patch

```diff
@@ -900,6 +898,8 @@ Signed-off-by: Tamas K Lengyel <lengyelt@ainfosec.com>
 -    pa += sizeof(uint64_t);
 -    tboot_copy_memory((unsigned char *)&sinit_mle_data, sizeof(sinit_mle_data),
 -                      pa);
++    /* map TXT heap into Xen addr space */
```

Comment:
Is this change specific to Xen 4.12 uprev or related to another ticket/issue?

Reply:
Specific to 4.12. Another contextual adjustment.